### PR TITLE
implement done command 15

### PR DESCRIPTION
**Pull request by unbekanntunity**
Changes
- Implemented done handler
- Added status enum
- Added a new property to the taskitem record (TaskStatus)

### CODE REVIEW

File: BusyList/Handlers/DoneHandler.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
+﻿using BusyList.Commands;
+using System;
+
+namespace BusyList.Handlers
+{
+    public class DoneHandler : IHandler<DoneCommand>
+    {
+        private readonly ITaskRepository _TaskRepository;
```
Comment (pobiega) [inline BusyList/Handlers/DoneHandler.cs:8]: Should be a lowercase t for this field (private fields follow the `_camelCase` rule).

File: BusyList/ITaskRepository.cs [hunk 2]
```diff
@@ -2,9 +2,16 @@
 
 namespace BusyList
 {
+    public enum TaskStatus
+    {
+        Done,
+        InProcess,
```
Comment (pobiega) [inline BusyList/ITaskRepository.cs:8]: InProgress?

File: BusyList/ITaskRepository.cs [hunk 3]
```diff
@@ -2,9 +2,16 @@
...
+        NotStarted
```
Comment (pobiega) [inline BusyList/ITaskRepository.cs:9]: This should be the first item, as the default enum value is in the first position, and it makes the most sense for NotStarted to be the default.

**Review by pobiega — CHANGES_REQUESTED** [review]

File: BusyList/ITaskRepository.cs (hunk 3 above)
Comment (unbekanntunity) [reply BusyList/ITaskRepository.cs:9]: Ohh yeah ofc

File: BusyList/ITaskRepository.cs [hunk 4]
```diff
@@ -2,9 +2,16 @@
 
 namespace BusyList
 {
+    public enum TaskStatus
+    {
+        NotStarted,
+        InPogress,
```
Comment (pobiega) [inline BusyList/ITaskRepository.cs:8]: InProgress

**Review by pobiega — CHANGES_REQUESTED** [review]

**Review by pobiega — APPROVED** [review]

**Review by 18-F-cali — APPROVED** [review]

**Pull request merged**